smartcontract/service/native/ont/states.go

```diff
@@ -82,7 +82,12 @@ func (this *State) Deserialize(r io.Reader) error {
 	var err error
 	this.From, err = utils.ReadAddress(r)
 	if err != nil {
-		return fmt.Errorf("[State] deserialize from error:%v", err)
+		//for array case, the first element is count
+		//so from is the second element
+		this.From, err = utils.ReadAddress(r)
```

Comment:
Can you elaborate when this will happen? ambiguous serialization format is not acceptable.